src/components/Analysis/AnalysisListItem.jsx

```diff
@@ -29,10 +52,10 @@ const ModeItem = ({ timeseriesSortedByMode, totalCO2 }) => {
     <>
       <ListItem className="u-pl-1-s u-pl-2" disabled={isDisabled}>
         <ListItemIcon>
-          <Avatar icon={pickModeIcon(mode)} color={modeToColor(mode)} />
+          <ItemIcon type={type} sortedTimeserieKey={sortedTimeserieKey} />
         </ListItemIcon>
         <ListItemText
-          primary={t(`trips.modes.${mode}`)}
+          primary={t(`trips.${type}.${sortedTimeserieKey}`)}
```

Comment:
it looks like you fixed some bug in this component. Let's add at least test on the added feature/fix